parse/DoubleComplexValueRefParser.cpp

```diff
@@ -48,7 +48,7 @@ namespace parse {
 
             species_empire_opinion
                 = (
-                    (  tok.SpeciesOpinion_ [ _a = construct<std::string>("SpeciesEmpireOpinion") ]
```

Comment:
Not related to CMake.  Please drop this from the PR.

Aside from that is a strange way of fixing this. I wouldn't pull it this way.